packages/medusa/src/repositories/discount-condition.ts

```diff
@@ -256,6 +256,10 @@ export class DiscountConditionRepository extends Repository<DiscountCondition> {
     //    if condition operation is `in` and the query for conditions defined for the given type is empty, the discount is invalid
     //    if condition operation is `not_in` and the query for conditions defined for the given type is not empty, the discount is invalid
     for (const condition of discountConditions) {
+      if (condition.type === DiscountConditionType.CUSTOMER_GROUPS) {
```

Comment:
comment (for reviewer): This repository method is responsible for checking the validity of a product, which is why we skip the customer groups conditions